doc/integrations/Skype.md

```diff
@@ -2,14 +2,14 @@
 Create a new file, `config.json` that has the following:
 ```
 {
-	"output": {
-		"skype": {
-			"username": "<skypeUsername/skypeEmail>",
-			"password": "<skypePassword>",
-			"commandPrefix": "!",
-			"conversations": ["<conversationID1>","<conversationID2",...]
-		}
-	}
+    "output": {
```

Comment:
please review all documentation that has been changed to see how these changes affect it